src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -49,6 +56,17 @@ const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
         </RouteTabs>
 
         <Box>{children}</Box>
+        {!route.displayFullPage && typeof window !== "undefined" && (
```

Comment:
Can remove `route.displayFullPage` -- we used that sometimes for when we have a route that complete takes over the view, but here it won't be used. 

FYI (if you don't already know) -- The `typeof window !== undefined` means: "the page has mounted on the client-side. On the server side render pass, when we statically render out html for SEO, don't render this rail. Why? Because recently viewed requires a logged in user! So its not relevant to a google bot.